distage/distage-framework/src/main/scala/izumi/distage/effect/modules/MonixDIEffectModule.scala

```diff
@@ -0,0 +1,11 @@
+package izumi.distage.effect.modules
+
+import distage.ModuleDef
+import izumi.distage.model.effect.DIEffectRunner
+import monix.bio
+
+object MonixDIEffectModule extends MonixDIEffectModule
+
+trait MonixDIEffectModule extends ModuleDef {
+  addImplicit[DIEffectRunner[bio.Task]]
```

Comment:
@VladPodilnyk  Just DIEffectRunner is not enough to run distage on monix, at least `DIEffect` & `DIEffectAsync` are also required (these can just use `cats` instance). Also the module should include all the cats instances, like `CatsDIEffectModule` (There's actually already a polymorphic implementation for `DIEffectModule` for Monix - https://github.com/BlueBrain/nexus/blob/cb8b4a587acbef7d0ad9d77240e4d6c30a6b20fe/cli/src/main/scala/ch/epfl/bluebrain/nexus/cli/EffectModule.scala we may reuse it and make `CatsDIEffectModule` polymorphic)

Reply:
Yup, I know that I need `DIEffect` and `DIEffectAsync` (that is the reason why the tests are broken). I am just thinking of reusing `cats` instance and you kindly pointed me in the right direction.